lib/pbench/server/database/alembic.run

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+# Alembic command expects to run in our database directory.
+cd lib/pbench/server/database
+
+# Our env requires a pbench-server config file.
+export _PBENCH_SERVER_CONFIG=$(pwd)/pbench-server.cfg
```

Comment:
Or we could create the actual `alembic.ini` file *here* from a template.

And why are we duplicating the config file? We're doing this after an install, so `/opt/pbench-server/lib/config/pbench-server.cfg` has the right settings and this copy doesn't.

Reply:
We can't create a migration revision in a running container.